ADOPTERS.md

```diff
@@ -30,3 +30,5 @@ This is a list of adopters of using PowerShell in production or in their product
   as well as supporting PowerShell Core in both Windows and Linux EC2 Images.
 * [Azure Resource Manager Deployment Scripts](https://docs.microsoft.com/azure/azure-resource-manager/templates/deployment-script-template) Complete the "last mile" of your Azure Resource Manager (ARM) template deployments with a Deployment Script, which enables you to run an arbitrary PowerShell script in the context of a deployment.
   Designed to let you complete tasks that should be part of a deployment, but are not possible in an ARM template today — for example, creating a Key Vault certificate or querying an external API for a new CIDR block. 
+* [Azure Pipelines Hosted Agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops) Agents used by Azure Pipelines customers have PowerShell pre-installed so that customers can make use of it for all their CI/CD needs.
```

Comment:
Links should not include the locale so that it uses the user one.  Should explicitly call out Windows, Linux, and macOS have PowerShell available by default.

```suggestion
* [Azure Pipelines Hosted Agents](https://docs.microsoft.com/azure/devops/pipelines/agents/hosted?view=azure-devops) Agents used by Azure Pipelines customers have PowerShell pre-installed on Windows, Linux, and macOS so that customers can make use of it for all their CI/CD needs.
```

Reply:
Addressed, thanks.